### ISSUE TITLE
Image from dls needs two denorms from imagenet_stats

### ISSUE THREAD

**Issue by Bunoviske** [fastai/fastai#3398]
In fastai version 2.3.0 and 2.3.1, I need to denorm (decode) the image two times so I can get the original one from learner.dls. In version 2.2.7 works fine. I also imagine what might happen during the training and get_preds. Maybe you are normalizing the image twice.

```
input, probabilities, groundTruth, decoded = learner.get_preds(dl=dl,  with_input=True, with_loss=False, with_decoded=True, act=F.softmax) 

rawImage = copy(input[0])

nrm = Normalize.from_stats(*imagenet_stats)
image = nrm.decode(rawImage) #fastai version 2.3.1 needs two denorms!!!! 
image = nrm.decode(image).clamp(0,1).squeeze().permute(1,2,0) # denorm image from imagenet stats
```

**Comment by Bunoviske** [comment]
First image: no denorm
Second image: one denorm
Third image: two denorms

![noDenorm](https://user-images.githubusercontent.com/16785926/120561808-8890be80-c405-11eb-8375-5e40b7c3c960.png)
![oneDenorm](https://user-images.githubusercontent.com/16785926/120561811-89295500-c405-11eb-87e8-e696a26c8249.png)
![twoDenorms](https://user-images.githubusercontent.com/16785926/120561803-87f82800-c405-11eb-88cc-e4e7674eeb38.png)

**Comment by riven314** [comment]
@Bunoviske 
Would you mind creating a minimal reproducible example in, lets say, a Colab notebook? 
I know there are some changes brought in fastai 2.3+ wrt adding normalization in validation set, which may be a cause for that.
I would be great if u have a notebook for me to verify that

**Comment by muellerzr** [comment]
Also: Try installing the dev version of fastai and fastcore and try again too just to verify:
```python
pip install git+https://github.com/fastai/fastai
pip install git+https://github.com/fastai/fastcore
```

**Comment by muellerzr** [comment]
@Bunoviske looks like this was resolved in the latest version (2.4) :) cc @jph00

**Comment by Bunoviske** [comment]
Nice! Sorry, i did not have time to test it. Thanks
